packages/thumbprint-react/utils/ConditionalPortal.jsx

```diff
@@ -1,21 +1,37 @@
+import React from 'react';
 import ReactDOM from 'react-dom';
 import PropTypes from 'prop-types';
 import canUseDOM from './can-use-dom';
 
 /**
- * Component to conditionally portal a component to the end of the <body> if a certain condition is
+ * Component to conditionally portal a component to the end of the `<body>` if a certain condition is
  * true. Also automatically guards against trying to use portals in SSR where `document` is not
  * defined.
  */
 export default function ConditionalPortal({ shouldDisplace, children }) {
-    return canUseDOM && shouldDisplace ? ReactDOM.createPortal(children, document.body) : children;
+    if (!children) {
+        return null;
+    }
+
+    return canUseDOM && shouldDisplace ? (
+        ReactDOM.createPortal(<React.Fragment>{children}</React.Fragment>, document.body)
+    ) : (
+        <React.Fragment>{children}</React.Fragment>
```

Comment:
Is `React.Fragment` needed? I think it can be just `children`.

Reply:
the fragment is here to get the linter working. it doesn't identify it as a React component if it has no JSX. See my comment in front-end-questions

Reply:
Not ideal to modify code for the linter, but it doesn't show up in the output, and it's better than a div. The alternative is not have lint errors like the proptypes warning show up, which is what caused this in the first place.